.gitignore

```diff
@@ -1,19 +0,0 @@
-.idea
```

Comment:
It seems reasonable to keep the gitignore in the branch, no? It still serves the same purpose in the new branch.

Reply:
Done